src/pages/manage/labels/index.page.tsx

```diff
@@ -0,0 +1,222 @@
+        accessor: 'name',
+        Cell: ({ cell }) => <Text>{cell.value}</Text>,
+      },
+      {
+        Header: t('labels.overview.table.invisible'),
+        accessor: 'invisible',
+        Cell: ({ cell }) =>
+          cell.value === 'invisible' ? (
+            <Text>{t('labels.overview.table.invisible')}</Text>
+          ) : null,
+      },
+      {
+        Header: t('labels.overview.table.private'),
+        accessor: 'private',
+        Cell: ({ cell }) =>
+          cell.value === 'private' ? (
+            <Text>{t('labels.overview.table.private')}</Text>
+          ) : null,
+      },
+      {
+        Header: t('labels.overview.table.excluded'),
+        accessor: 'excluded',
+        Cell: ({ cell }) =>
+          cell.value === true ? (
+            <Text>{t('labels.overview.table.excluded')}</Text>
+          ) : null,
+      },
+      {
+        Header: t('labels.overview.table.options'),
+        accessor: 'options',
+        Cell: ({ cell }) => (
+          <Link href={'/manage/labels/' + cell.value}>
+            {t('labels.overview.table.edit')}
+          </Link>
+        ),
+      },
+    ],
+    [t],
+  );
+  const labels: Label[] = useMemo(
+    () => labelsQuery.data?.member ?? [],
+    [labelsQuery.data?.member],
+  );
+
+  const handleInputSearch = useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      const searchTerm: string = event.target.value.toString().trim();
+      setCurrentPageLabels(1);
+      setSearchInput(searchTerm);
+    },
+    [],
+  );
+
+  return (
+    <Page>
+      <Page.Title>{t('labels.overview.title')}</Page.Title>
+      <Page.Content spacing={5}>
+        <FormElement
+          id="labels-overview-search"
+          label={t('labels.overview.search.label')}
+          Component={
+            <Input
+              placeholder={t('labels.overview.search.placeholder')}
+              onChange={debounce(handleInputSearch, 275)}
+            />
+          }
+        />
+
+        <Stack spacing={5}>
+          {labelsQuery.status === QueryStatus.LOADING ? (
+            <Spinner />
+          ) : (
+            <Stack
+              backgroundColor="white"
+              padding={4}
+              borderRadius={getGlobalBorderRadius}
+              spacing={5}
+              css={`
+                box-shadow: ${getGlobalValue('boxShadow.medium')};
+              `}
+            >
+              {labelsQuery.status === QueryStatus.SUCCESS &&
+              labels.length > 0 ? (
+                <Stack spacing={3}>
+                  <Inline
+                    forwardedAs="div"
+                    width="100%"
+                    alignItems="center"
+                    spacing={5}
+                  >
+                    <Inline spacing={3}>
+                      {actions.map(({ iconName, title, onClick, disabled }) => (
+                        <Button
+                          key={title}
+                          variant={ButtonVariants.SECONDARY}
```

Comment:
I would opt for the "Primary" variant here.
```suggestion
                          variant={ButtonVariants.PRIMARY}
```
<img width="1344" height="689" alt="Screenshot 2025-09-15 at 15 07 42" src="https://github.com/user-attachments/assets/ad298281-0ea4-47b6-a805-41b56ee84db7" />